index.html

```diff
@@ -17,6 +17,10 @@
     <link rel="stylesheet" href="./assets/css/main.css">
     <!-- fonts -->
     <link rel="preconnect" href="https://fonts.gstatic.com">
+    <!--scroll up button-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/css?family=Merriweather:400,900,900i" rel="stylesheet">
+    
```

Comment:
Since we already have font-awesome (see above) and the font isn't being used these can be removed.